quarkus/integration-tests/server/src/test/resources/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM registry.access.redhat.com/ubi8/ubi-minimal
+FROM registry.access.redhat.com/ubi9/ubi-minimal
```

Comment:
We actually use ubi8 for the upstream images, as we needed to baseline the JDK based image on that for Openshift compatibility. That being said, I should probably upgrade the cli and server-native images to ubi9 as we don't have similar downstream considerations.

Reply:
Sorry, that's a leftover from running it on my local box which has a much newer glibc than CI.